### PR TITLE
Add content to why makers academy

### LINKED ISSUE #384: 'About us' section addition
In 'why Makers Academy' I'd like to add another pair:

Job support - our service isn't complete until you're hired

Community of learners - because it's easier when you aren't alone

Can these go between the top line and the middle line please?

### PR DESCRIPTION
**Pull request by nikeshashar**
Add two more benefits as requested by @jordanpoulton. Closes #384

### CODE REVIEW

**Comment by nikeshashar** [comment]
The dashes look weird. 

image

File: source/partials/_benefits_table.html.haml [hunk 1]
```diff
@@ -9,8 +9,11 @@
             %td Learning cutting edge coding techniques such as agile &amp; TDD
             %td Located at the heart of Silicon Roundabout with a big work space
           %tr
+            %td Job support - our service isn't complete until you're hired
```
Comment (spike01) [inline source/partials/_benefits_table.html.haml]: Use &mdash;

Comment (spike01) [inline source/partials/_benefits_table.html.haml]: That's `&mdash;`

Comment (spike01) [inline source/partials/_benefits_table.html.haml]: Actually maybe an `&ndash;` might look better - try both

**Comment by spike01** [comment]
We're also not at the heart of Silicon Roundabout any more - maybe switch it to the heart of Tech City?
Otherwise :shipit: pending dashes

**Pull request merged**